modules/wss4j/src/org/apache/ws/security/processor/EncryptedKeyProcessor.java

```diff
@@ -226,17 +226,8 @@ else if (secRef.containsKeyIdentifier()) {
                     X509Certificate[] certs = null;
 					if (WSConstants.WSS_SAML_KI_VALUE_TYPE.equals(secRef
 							.getKeyIdentifierValueType())) {
-						Element token = secRef.getKeyIdentifierTokenElement(
-								doc, docInfo, cb);
-
-						if (crypto == null) {
-							throw new WSSecurityException(
-									WSSecurityException.FAILURE,
-									"noSigCryptoFile");
-						}
-						SAMLKeyInfo samlKi = SAMLUtil.getSAMLKeyInfo(token,
-								crypto, cb);
-						certs = samlKi.getCerts();
+                        throw new WSSecurityException(
+                                WSSecurityException.FAILED_CHECK, "SAML 1.x is not supported");
```

Comment:
Looks likes the indentation is wrong. May be due to using tab character in old code. Fix to use 4 spaces instead.

Reply:
fixed in 2f696f6746f53ec59ed149ea7753dbcb09f7b044